packages/core/strapi/lib/services/entity-service/types/params/pagination.d.ts

```diff
@@ -0,0 +1,11 @@
+export type PageNotation = {
+  page?: number;
+  pageSize?: number;
+};
+
+export type OffsetNotation = {
+  start?: number;
+  limit?: number;
+};
+
+export type Any = PageNotation | OffsetNotation;
```

Comment:
```suggestion
export type AnyNotation = PageNotation | OffsetNotation;
```

Reply:
If we change to `AnyNotation` here, we should do the same in the sort.
What makes you prefer over the other @innerdvations ?

Reply:
Oh, I see, thanks for the explanation